environments/minikube/jupyterhub/values.yaml.gotmpl

```diff
@@ -2,7 +2,8 @@ singleuser:
   extraEnv:
     AWS_ACCESS_KEY_ID: admin
     AWS_SECRET_ACCESS_KEY: password123
-    KAMU_JWT_SECRET: secret123
+    # Keep in sync with config.auth.jwtSecret in kamu-api-server
+    KAMU_JWT_SECRET: secret1234567890
```

Comment:
I think you need to bump jupyter image to `0.14.0`. After that sharing JWT secret with jupyter will no longer be necessary as @zaychenko-sergei changed it to use a [non-interactive login](https://github.com/kamu-data/kamu-cli/blob/master/images/demo/jupyter/kamu-start-hook.sh#L7) via GH access token.

Reply:
Old image will not work as it will generate an incompatible JWT token. Make sure to test the resulting token by logging in into you local Jupyter and doing `kamu login --check`.

Reply:
```shell
# 1
(base) jovyan@jupyter-s373r:~$ kamu login oauth --user github "${GITHUB_TOKEN}" "${KAMU_NODE_URL#odf+}"
Access token valid: http://kamu-api-server:8080/

# 2
(base) jovyan@jupyter-s373r:~$ kamu login --check
Error: No access token found for: https://platform.demo.kamu.dev/

# 3
(base) jovyan@jupyter-s373r:~$ kamu login --check http://kamu-api-server:8080/
Error: No access token found for: http://kamu-api-server:8080/
```

Reply:
```
(base) jovyan@jupyter-s373r:~$ cat .kamutokenstore
```
```yaml
kind: KamuTokenStore
version: 1
content:
- frontendUrl: null # <--- ?
  backendUrl: http://kamu-api-server:8080/
  tokens:
  - accountName: kamu
    accessToken: <TOKEN>
```

Reply:
I've tried some more different options, so far no luck.

Reply:
At the moment, I think the reason is that the search is by `frontendUrl`:

```rust
impl Command for LoginCommand {
    // ...
    async fn run(&mut self) -> Result<(), CLIError> {
        // ...
        // Check token and exit
        if self.check {
            return if let Some(token_find_report) = self
                .access_token_registry_service
                .find_by_frontend_url(self.scope, &odf_server_frontend_url)
            {
                // ...
            } else {
                Err(CLIError::usage_error(format!(
                    "No access token found for: {odf_server_frontend_url}",
                )))
            };
        }
    // ...
```

`null` !== `http://kamu-api-server:8080/ (odf_server_frontend_url)`

Reply:
I guess this doesn't really apply to the scope of the current PR, I suggest merge it

Reply:
Not actual -- fixed via:
- https://github.com/kamu-data/kamu-cli/pull/641

BUT, оupiter demo image has not yet been rebuilt

Reply:
The image build in progress...

Reply:
Done